sdk/rust/reference.mdx

```diff
@@ -1,4 +1,275 @@
 ---
 title: Reference
-url: https://docs.rs/libsql
+description: libSQL Rust Reference
 ---
+
+The libSQL Rust crate contains everything you need to work with Turso and asynchronous Rust.
+
+## Installing
+
+Install the package in your project using the following command:
+
+```bash
+cargo add libsql
+```
+
```

Comment:
```suggestion

### Conditional compilation

The libsql rust client supports conditionally compiling certain features to reduce compile times depending on what features you would like to use. Currently, all features are enabled by default but it is possible to set `default-features = false` and to enable only the ones you would like. The available features are `remote` (http only client), `core` (local database only), `replication` (embedded replica features) and `encryption` (encryption at rest support, this is disabled by default). For more information on how to enable features you checkout the [cargo book](https://doc.rust-lang.org/cargo/reference/features.html#dependency-features).

```

Reply:
@LucioFranco Would this example be sufficient here on how to customise this?

```toml
[dependencies]
libsql = { version = "...", "default-features" = false, features = ["encryption"] }
```